graylog2-web-interface/src/views/components/sidebar/FieldList.jsx

```diff
@@ -18,16 +20,16 @@ const isReservedField = (fieldName) => MessageFieldsFilter.FILTERED_FIELDS.inclu
 
 const FieldList = createReactClass({
```

Comment:
Now that the mixins are gone, can we turn this into a class-based or functional component?

Reply:
I thought about it as well, but I prefer doing this in another PR, otherwise reviewing the `FieldList` changes would have been more complicated.